File: docs/v1/examples/guides/install/maven/src/main/java/kamon/example/Start.java
package kamon.example;

import kamon.Kamon;
import kamon.prometheus.PrometheusReporter;
import kamon.zipkin.ZipkinReporter;

public class Start {

  // tag:load-modules:start
  public static void main(String[] args) {
    Kamon.addReporter(new PrometheusReporter());
    Kamon.addReporter(new ZipkinReporter());
  }
  // tag:load-modules:end
}

File: docs/v1/examples/guides/install/gradle/src/main/kotlin/kamon/example/Start.kt
package kamon.example

import kamon.Kamon
import kamon.prometheus.PrometheusReporter
import kamon.zipkin.ZipkinReporter

// tag:load-modules:start
fun main(args: Array<String>) {
  Kamon.addReporter(PrometheusReporter())
  Kamon.addReporter(ZipkinReporter())
}
// tag:load-modules:end